babel/src/test/java/org/apache/calcite/test/BabelParserTest.java

```diff
@@ -328,6 +328,54 @@ private void checkParseInfixCast(String sqlType) {
     sql(sql).ok(expected);
   }
 
+  @Test public void testTableAttributeJournal() {
```

Comment:
I would suggest adding "Default" in the test name so it is more clear that this test the default behavior of Journal. (same for the next two tests).

Reply:
Sure I could add default to that test

Reply:
Well actually, since there are two things to be specified when you test the attribute "Journal", it might be clearer to point out what combination of JournalAttribute and JournalType you are trying to test in the test name. If you agree with me, then most of the test names are subject to change.

Reply:
How do you mean? For the journal modifiers, I have the modifier keyword on the end and the "local", "not local" only work with the "after" keyword so I didn't specify the type there. Is there another naming scheme you have in mind to make it clearer?

Reply:
I would make comments in the newest version to clarify my suggestion